xml/docu_styleguide.changes.xml

```diff
@@ -6,12 +6,23 @@
 ]>
 <appendix xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="app-change">
 <!-- Add new versions to the beginning of the list. -->
- <title>List of Changes to the Style Guide</title>
+ <title>List of changes to the style guide</title>
  <info/>
  <para>
   This section lists the changes between versions of this Style Guide.
  </para>
 <!-- Add new versions to the beginning of the list. -->
+<sect1 xml:id="sec-change-2020-12">
+ <title>Changes 2020-12</title>
+ <itemizedlist>
+  <listitem>
+   <para>
+     Updated the Style Guide to recommend the use of sentence case titles.
```

Comment:
I guess it should be the following now?:

```suggestion
       Updated the Style guide to recommend the use of sentence case titles.
```

Reply:
I think we hadn't really spoke abotu whether or not guide titles would remain as title case. I, personally, think we should keep it as Style Guide as opposed to Style guide (to me, that just looks like the title is "Style" which is... weird)

Any thoughts on this, @lproven @taroth21 @Loquacity @cwickert ?

Reply:
If we keep title case for book titles, we probably need to continue to explain it in the style guide. Hence, I'd vote for just throwing it out entirely. 

(Our product names will continue title-cased but they come straight from marketing anyway, so I don't think that will be an issue, we just need to capitalize them the same way that marketing does.)

Reply:
I mean, I would be quite happy to include a note that says the guide titles are continued in title case - it's not a hard thing to explain.

Reply:
I agree. Book titles not in title case looks a bit weird to me. Sections / subsections in sentence case looks fine.

Reply:
I agree. Book titles in sentence case look strange. And as opposed to titles within the text that also show up in contributors' PRs, I have seldomly (if ever) come across a PR where someone wanted to change the book title. In addition, I would say changing the book titles is entirely up to us. So it would be no problem for us to keep book titles in title case.

Reply:
Ok, I guess everyone hates consistency ^^

>I mean, I would be quite happy to include a note that says the guide titles are continued in title case - it's not a hard thing to explain.

What I mean is that we currently have an explanation of title case in the document and if we continue to use title case for some thing, like (titles of books and I assume sets & articles as well), that explanation should probably be kept. Otherwise, people will be confused how to do it (e.g. Germans will generally like the fact that they are allowed to capitalize nouns but will be confused by capitalizing verbs too; then you explain that you need to capitalize most words ... and people will capitalize everything including short prepositions ...). I.e. if we continue to need title case, please don't remove the explanation from the guide. Especially as everyone does it a little differently too, e.g. with some people capitalizing four/five-letter prepositions and others not doing so.

I also think that if we progressively move into the Eppo world, the average book title will become less relevant anyway. And in fact, this inconsistency may hinder reuse at some point.

And finally, if we apply the the same title rule to articles that we apply to books, Meike is getting (figurative) loads of PRs for new SBPs articles, so I don't think the argument quite applies.

Reply:
We hate consistency and kittens, it's just how we roll ;)